src/app/reader/reader.component.ts

```diff
@@ -276,8 +276,14 @@ export class ReaderComponent implements OnInit, OnDestroy {
     this.bookmarManagerService.identifier = data.id!;
     this.bookmarManagerService.el.hidden = true;
 
+    const urls: Array<string> = [];
+
     for (const [key, value] of Object.entries(blobs)) {
-      elementHtml = elementHtml.replaceAll(`ttu:${key}`, URL.createObjectURL(value));
+      const url = URL.createObjectURL(value);
+      urls.push(url);
+      elementHtml = elementHtml.
+        replaceAll(`data:image/gif;ttu:${key};base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==`, url).
```

Comment:
Since it's used multiple times, probably a good idea to have a utility function (in utils dir) that exports a function which returns in this format, so we don't have to keep track of the value `R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==`

Reply:
will add an exported function in the html-fixer util to return the full data uri for usage in build places